ds-prep-python-functions/main.py

```diff
@@ -0,0 +1,18 @@
+bool_to_int = lambda value: int(value)
+print(bool_to_int(True))
+get_smaller = lambda a , b: min(a , b)
+print(get_smaller(5 , 10))
+def cube(x): return x**3
+print(cube(5))
+def absolute_difference(a , b): return (a - b)
+print(absolute_difference(5 , 3))
+def squared_difference(a , b): return ((a - b)**2)
+print(squared_difference(3 , 2))
+def hours_to_minutes(hours): return (hours*60)
+print(hours_to_minutes(2))
+def get_circumference(radius): print(radius*2*3.14)
+get_circumference(radius = 2)
+def linear_transform(x , slope , intercept): return ((slope*x) + intercept)
+print(linear_transform(x = 2 , slope = 3 , intercept = 2))
+def standardize(x , x_center , scale_size): print((x - x_center) / scale_size)
+standardize(8 , 2 , 2)
```

Comment:
Call each of these functions at least twice, using keyword arguments, with a different argument order in each call:

---

![image](https://user-images.githubusercontent.com/16143771/148816309-c0344e76-2e59-4429-bc26-b02bd93402aa.png)